### PR TITLE
[feat] New example `Belle-Llama2-13B-GGUF`

### PR DESCRIPTION
**Pull request by apepkuss**

### CODE REVIEW

**Comment by juntao** [comment]
Hello, I am a [code review bot](https://github.com/flows-network/github-pr-summary/) on [flows.network](https://flows.network/).

It could take a few minutes for me to analyze this PR. Relax, grab a cup of coffee and check back later. Thanks!

**Comment by apepkuss** [comment]
@hydai Please help review this PR. Thanks a lot!

File: wasmedge-ggml-belle-chat/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[package]
+name = "wasmedge-ggml-belle-chat"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+chat-prompts = "0.1"
+endpoints = "0.1"
+wasi-nn = { git = "https://github.com/second-state/wasmedge-wasi-nn", branch = "dm4/ggml" }
```
Comment (hydai) [inline wasmedge-ggml-belle-chat/Cargo.toml:9]: @dm4 
Please check if we are still on this branch?

**Review by hydai — CHANGES_REQUESTED** [review]

File: wasmedge-ggml-belle-chat/Cargo.toml (hunk 1 above)
Comment (dm4) [reply wasmedge-ggml-belle-chat/Cargo.toml:9]: Please use the official `ggml` branch instead. Thank you!

**Review by hydai — CHANGES_REQUESTED** [review]

File: wasmedge-ggml-belle-chat/Cargo.toml (hunk 1 above)
Comment (apepkuss) [reply wasmedge-ggml-belle-chat/Cargo.toml:9]: Fixed. Thanks!

**Comment by apepkuss** [comment]
@hydai @dm4 The review issues has been fixed. Please review the changes. Thanks a lot!

**Review by hydai — APPROVED** [review]

**Pull request merged**